src/LoopbackModelBase.js

```diff
@@ -0,0 +1,103 @@
+const MicroserviceError = require('./MicroserviceError');
+
+module.exports = class LoopbackModelBase {
+    constructor({ modelName, model }) {
+        if (!model) {
+            throw LoopbackModelBase.createError(`No Loopback Model Instance was given to the
+                LoopbackModelBase constructor`);
+        }
+
+        this.modelName = modelName || model.modelName;
+        this[this.modelName] = model;
```

Comment:
why are you storing this under a dynamic name and not a static one? `this.loopbackModel` would be fine, not?